airflow/config_templates/config.yml

```
@@ -1779,6 +1779,15 @@ webserver:
       type: string
       example: "sha256"
       default: "md5"
+    skip_trigger_form_if_no_params:
```

```suggestion
    show_trigger_form_if_no_params:
```
Will avoid a double negation in the code such as `not cong.getboolean(...)`. A bit easier to read IMO